### PR TITLE
Deleted the real-time configuration option

### PR DESCRIPTION
**Pull request by mattclarke**
Deleted the real-time configuration button and related code as that is not relevant to the ESS.

### CODE REVIEW

File: nicos_ess/loki/gui/cmdlets.py [hunk 1]
```diff
@@ -122,25 +120,12 @@ def on_middleLoop_currentIndexChanged(self, mloop):
 
     def on_client_experiment(self, data):
```
Comment (umithardal) [inline nicos_ess/loki/gui/cmdlets.py:121]: So there is nothing left in this function? May be we should remove it altogether?

File: nicos_ess/loki/gui/cmdlets.py [hunk 2]
```diff
@@ -122,25 +120,12 @@ def on_middleLoop_currentIndexChanged(self, mloop):
...
 
-    @pyqtSlot()
-    def on_rtConfBtn_clicked(self):
-        dlg = RtConfigDialog(self)
-        dlg.setSettings(self.rt_settings)
-        if not dlg.exec_():
-            return
-        self.rt_settings = dlg.getSettings()
-
     def getValues(self):
```
Comment (umithardal) [inline nicos_ess/loki/gui/cmdlets.py:129]: This function returns an empty dict, do we need this?

File: nicos_ess/loki/gui/cmdlets.py (hunk 1 above)
Comment (mattclarke) [reply nicos_ess/loki/gui/cmdlets.py:121]: It calls `self._clearTable()` see below

File: nicos_ess/loki/gui/cmdlets.py (hunk 2 above)
Comment (mattclarke) [reply nicos_ess/loki/gui/cmdlets.py:129]: Not relevant to this PR, but I'll remove it anyway ;)

**Pull request merged**